src/java_bytecode/java_class_loader.cpp

```diff
@@ -74,38 +83,76 @@ void java_class_loadert::set_java_cp_include_files(
   java_cp_include_files=_java_cp_include_files;
 }
 
-java_bytecode_parse_treet &java_class_loadert::get_parse_tree(
+/// Retrieves a class file from a jar and loads it into the tree
+bool java_class_loadert::get_class_file(
```

Comment:
Why does the class loader need to be rewritten when the commit message says this is about putting stuff in a library file?

Reply:
the class loader needs to load from the internal file buffer